mobile/lib/common/status_screen.dart

```diff
@@ -20,39 +21,78 @@ class _StatusScreenState extends State<StatusScreen> {
     ServiceStatusNotifier serviceStatusNotifier = context.watch<ServiceStatusNotifier>();
 
     final orderbookStatus =
-        statusToString(serviceStatusNotifier.getServiceStatus(Service.Orderbook));
+        serviceStatusToString(serviceStatusNotifier.getServiceStatus(Service.Orderbook));
     final coordinatorStatus =
-        statusToString(serviceStatusNotifier.getServiceStatus(Service.Coordinator));
-    final overallStatus = statusToString(serviceStatusNotifier.overall());
+        serviceStatusToString(serviceStatusNotifier.getServiceStatus(Service.Coordinator));
+    final overallStatus = serviceStatusToString(serviceStatusNotifier.overall());
+
+    ChannelStatusNotifier channelStatusNotifier = context.watch<ChannelStatusNotifier>();
+
+    final channelStatus = channelStatusToString(channelStatusNotifier.getChannelStatus());
+
+    var widgets = [
+      Padding(
+          padding: const EdgeInsets.all(32.0),
+          child: Column(
+            children: [
+              const SizedBox(height: 20),
+              ValueDataRow(
+                type: ValueType.text,
+                value: overallStatus,
+                label: "Services",
+              ),
+              const Divider(),
+              ValueDataRow(
+                type: ValueType.text,
+                value: orderbookStatus,
+                label: "Orderbook",
+              ),
+              const SizedBox(height: 10),
+              ValueDataRow(
+                type: ValueType.text,
+                value: coordinatorStatus,
+                label: "LSP",
+              ),
+            ],
+          )),
+      Padding(
+          padding: const EdgeInsets.all(32.0),
+          child: Column(
+            children: [
+              const SizedBox(height: 20),
+              ValueDataRow(
+                type: ValueType.text,
+                value: channelStatus,
+                label: "Channel status",
+              ),
+            ],
+          )),
+    ];
+
+    if (channelStatusNotifier.isClosing()) {
+      widgets.add(Padding(
+          padding: const EdgeInsets.all(32.0),
+          child: RichText(
+              text: const TextSpan(style: TextStyle(color: Colors.black, fontSize: 18), children: [
+            TextSpan(
+                text: "Your channel with 10101 is being closed on-chain!\n\n",
+                style: TextStyle(fontWeight: FontWeight.bold)),
+            TextSpan(
+                text:
+                    "Your Lightning funds will return back to your on-chain wallet after some time. You will have to reopen the app at some point in the future so that your node can claim them back.\n\n"),
+            TextSpan(
+                text:
+                    "If you had a position open your payout will arrive in your on-chain wallet soon after the expiry time. \n")
+          ]))));
+    }
```

Comment:
🙃 I like extraction of the widgets, but could we please make it into a function instead of variable? then it's easier to find it in the IDE

Reply:
Do you mind refactoring this yourself? I'm not sure I understand what function you want to define here. I only extracted it so that we could add the last widget depending on the channel status.

Reply:
no probs! will do that as a follow-up